src/main/java/org/carlmontrobotics/robotcode2023/RobotContainer.java

```diff
@@ -168,4 +184,7 @@ private double inputProcessing(double value) {
   private Trigger axisTrigger(Joystick stick, Axis axis) {
     return new Trigger(() -> Math.abs(getStickValue(stick, axis)) > MIN_AXIS_TRIGGER_VALUE);
   }
+  private Trigger axisTrigger(XboxController stick, Axis axis) {
+    return new Trigger(() -> Math.abs(getStickValue(stick, axis)) > MIN_AXIS_TRIGGER_VALUE);
+  }
```

Comment:
Instead of having to create an entire new method for every different controller, use `GenericHID` instead since both the `Joystick` class and `XboxController` class both extend `GenericHID`.